lib/shipping_agent/github/notification.rb

```diff
@@ -20,6 +21,34 @@ def update(status, description, deploy)
 
       private
 
+      # Github has an option to do this automaticly, but it only works
+      # for non-production environments.
+      def mark_old_deployments_inactive(deploy)
+        active_old_deployments(deploy).each { |deployment| mark_inactive(deployment) }
+      end
+
+      def active_old_deployments(current)
+        old_deployments(current).reject do |deployment|
+          current.url == deployment.url ||
+            deployment.rels[:statuses].get.data.any? { |s| s.state == "inactive" }
```

Comment:
Hmm, true - I'm confusing my "deployments".  I guess the confusing thing is that there is a `github` deployment and a `kubernetes` deployment.  Particularly because there's a `deploy` variable passed around in this class which is actually a `kubernetes` deployment right?  I think `mark_old_deployments_inactive` could benefit from named parameters here, so it's called with `mark_old_deployments_inactive(new_deploy: deploy)`.

Could be worth having a clearer distinction on terminology when we next change this file, as this is merged there's probably no point changing it now.

Also, a comment in the code would probably be good because I think this one-liner is a bit difficult to understand without a wider context of how we use github "deployments" (and therefore what the importance of marking them "inactive" is).